cmd/cdn-cron/main.go

```diff
@@ -21,6 +23,8 @@ func main() {
 	logger := lager.NewLogger("cdn-cron")
 	logger.RegisterSink(lager.NewWriterSink(os.Stderr, lager.INFO))
 
+	rand.Seed(time.Now().UnixNano())
```

Comment:
This isn't necessary.

Reply:
Can you explain why? I am not super experienced with go but my understanding is this is a psuedo-random number generator and therefore it needs to be seeded or it will produce results in the same order upon restarts.

Reply:
Leaving this as is per the `Example (Rand)` section in the docs: https://golang.org/pkg/math/rand/